notebooks/demo/mg_property_graph.ipynb

```diff
@@ -0,0 +1,217 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Property Graph using Multiple GPU\n",
+    "#### Author : Don Acosta\n",
+    "\n",
+    "In this notebook, we will show how to create and use a Property Graph using multiple GPUs \n",
+    "\n",
+    "This notebook was developed and tested using RAPIDS 23.02 and CUDA 11.5. Please be aware that your system may be different, and you may need to modify the code or install packages to run the below examples. If you think you have found a bug or an error, please file an issue in [cuGraph](https://github.com/rapidsai/cugraph/issues)\n",
+    "\n",
+    "\n",
+    "CuGraph's multi-GPU features leverage Dask. RAPIDS has other projects based on Dask such as dask-cudf and dask-cuda. These products will also be used in this example. Check out [RAPIDS.ai](https://rapids.ai/) to learn more about these technologies."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Multi-GPU Property Graph\n",
+    "### Basic setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import needed libraries. We recommend using the [cugraph_dev](https://github.com/rapidsai/cugraph/tree/branch-23.02/conda/environments) env through conda\n",
+    "from dask.distributed import Client, wait\n",
+    "from dask_cuda import LocalCUDACluster\n",
+    "from cugraph.dask.comms import comms as Comms\n",
+    "import cugraph.dask as dask_cugraph\n",
+    "import cugraph\n",
+    "import dask_cudf\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import urllib.request\n",
+    "import os\n",
+    "\n",
+    "data_dir = '../data/'\n",
+    "if not os.path.exists(data_dir):\n",
+    "    print('creating data directory')\n",
+    "    os.system('mkdir ../data')\n",
+    "\n",
+    "# download the Hollywood dataset\n",
+    "base_url = 'https://rapidsai-data.s3.us-east-2.amazonaws.com/cugraph/benchmark/'\n",
+    "fn = 'hollywood.csv'\n",
+    "comp = '.gz'\n",
+    "\n",
+    "if not os.path.isfile(data_dir+fn):\n",
+    "    if not os.path.isfile(data_dir+fn+comp):\n",
+    "        print(f'Downloading {base_url+fn+comp} to {data_dir+fn+comp}')\n",
+    "        urllib.request.urlretrieve(base_url+fn+comp, data_dir+fn+comp)\n",
+    "    print(f'Decompressing {data_dir+fn+comp}...')\n",
+    "    os.system('gunzip '+data_dir+fn+comp)\n",
+    "    print(f'{data_dir+fn+comp} decompressed!')\n",
+    "else:\n",
+    "    print(f'Your data file, {data_dir+fn}, already exists')\n",
+    "\n",
+    "# File path, assuming Notebook directory\n",
+    "input_data_path = data_dir+fn"
```

Comment:
I think it would be much better if you could either add hollywood to `cugraph.experimental.datasets`, or use a different dataset already in `cugraph.experimental.datasets`, since it would allow you to replace all this code with 2 lines, would handle skipping the download if it was previously downloaded before, etc.

Reply:
per our discussion. this will be removed when datasets supports decompression and dask df creation. I will create an issue for that